File: Set-21-2021/C/ex002.c
/*
    Calcular a média de 5 notas de um aluno
    Exibir se ele está aprovado (media >= 6)
    Se ele está em recuperação (3 >= media < 6)
    Se ele está reprovado (media < 3)
*/

#include <stdio.h>

int main()
{
    int i;
    float media, nota, soma;

    soma = 0;
    
    for (i=0; i<5; i++ )
    {
        printf("Digite a %dº nota do aluno: ", i + 1);
        scanf("%f", &nota);
        soma = soma + nota;
    }
    media = soma / 5;
    if (media >= 6){
        printf("\nAluno aprovado com média %.2f", media);
    }
    else if (media >= 3 && media < 6){
        printf("\nAluno em recuperação com média %.2f", media);
    }
    else{
        printf("\nAluno reprovado com média %.2f", media);
    }
}

File: C C++/Set-21-2021/Aula/Python/ex002.py
# Calcular a média de 5 notas de um aluno
#    Exibir se ele está aprovado (media >= 6)
#    Se ele está em recuperação (3 >= media < 6)
#    Se ele está reprovado (media < 3)

i = 0
soma = 0

while i < 5:
    nota = float(input("Digite a {}° nota do aluno: ".format(i+1)))
    soma = soma + nota
    i = i + 1
media = soma / 5

if (media >= 6):
    print("\nAluno aprovado com media {}".format(media))
elif (media >= 3 and media < 6):
    print("\nAluno em recuperação com media {}".format(media))
else:
    print("\nAluno reprovado com media {}".format(media))
